### PR TITLE
Added About Page 

### PR DESCRIPTION
**Pull request by apluscs**
Long lines will be formatted after the content gets approved.

Screenshot 2020-07-30 at 9 26 55 AM

### CODE REVIEW

File: src/main/webapp/about.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        chats in order to encapsulate the true feeling and pace of a technical interview. We encapsulate the diversity of roles in the tech industry with our nine possible job positions ranging from Software Engineer to Technical Sales. 
+      </p>
+      
+      <h5>How It Works</h5>
+      <p>
+        First, register for an account on our site. After registering, you can play both the role of the interviewer or interviewee. 
+      </p> 
+      <h6>As an Interviewer</h6>
+      <p>
+        Be sure to update what positions you are qualified to give an interview for in your profile. This will be critical in matching you up with the corresponding interviewees.
```
Comment (meagancheese) [inline src/main/webapp/about.html]: matching you with corresponding interviewees.

File: src/main/webapp/about.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        <br>
+        We will use your availabilities specified on the <a href="availability.html">Availability Page</a> to schedule your interviews.
+        <br>
+        Interviewees will see your company and job when searching, so please update those accordingly. Once an interviewee is matched with you, you will get more details in an email. The interviews will be hosted on Google Meet.
+      </p> 
+      <h6>As an Interviewee</h6>
+      <p>
+        To search for an interview, navigate to the <a href="search-interview.html">Scheduling Page</a>. You can choose what position you're interested in and see what times and interviewers are available. You will be able to see their company and job, but no more personal information.
+        <br>
+        Once an interviewee is matched with you, you will get more details in an email. The interviews will be hosted on Google Meet.
```
Comment (meagancheese) [inline src/main/webapp/about.html]: interviewer, not interviewee

File: src/main/webapp/about.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      <h4>Our Mission</h5>
+      <p>
+        InterviewMe is a platform for conducting mock technical interviews where interviewers and 
+        interviewees alike can practice their skills. We aim to connect users with live video
+        chats in order to encapsulate the true feeling and pace of a technical interview. We encapsulate the diversity of roles in the tech industry with our nine possible job positions ranging from Software Engineer to Technical Sales. 
+      </p>
+      
+      <h5>How It Works</h5>
+      <p>
+        First, register for an account on our site. After registering, you can play both the role of the interviewer or interviewee.
```
Comment (meagancheese) [inline src/main/webapp/about.html]: This becomes irrelevant after they register. Maybe conditionally remove it based on if they're logged in and registered.

**Review by meagancheese — CHANGES_REQUESTED** [review]
Some small changes.

File: src/main/webapp/about.html (hunk 3 above)
Comment (apluscs) [reply src/main/webapp/about.html]: Removed that line.

**Review by meagancheese — APPROVED** [review]

**Review by GrantComm — APPROVED** [review]

File: src/main/webapp/about.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+          </div>
+        </div>
+      </nav>
+      <br>
+      <h1 class="text-center">About InterviewMe</h1>
+      <h4>Our Mission</h5>
+      <p>
+        InterviewMe is a platform for conducting mock technical interviews where interviewers and 
+        interviewees alike can practice their skills. We aim to connect users with live video
+        chats in order to encapsulate the true feeling and pace of a technical interview. We encapsulate the diversity of roles in the tech industry with our nine possible job positions ranging from Software Engineer to Technical Sales.
```
Comment (jalexanderqed) [inline src/main/webapp/about.html]: Break this line

Comment (jalexanderqed) [inline src/main/webapp/about.html]: "Encapsulate" is used twice. Maybe switch the second one to "support"?

File: src/main/webapp/about.html [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+      <h4>Our Mission</h5>
+      <p>
+        InterviewMe is a platform for conducting mock technical interviews where interviewers and 
+        interviewees alike can practice their skills. We aim to connect users with live video
+        chats in order to encapsulate the true feeling and pace of a technical interview. We encapsulate the diversity of roles in the tech industry with our nine possible job positions ranging from Software Engineer to Technical Sales. 
+      </p>
+      
+      <h5>How It Works</h5>
+      <p>
+        After registering on InterviewMe, you can play both the role of the interviewer or interviewee.
```
Comment (jalexanderqed) [inline src/main/webapp/about.html]: "both ... or ..." doesn't really make sense. Probably switch either both->either or or->and.

File: src/main/webapp/about.html [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+        chats in order to encapsulate the true feeling and pace of a technical interview. We encapsulate the diversity of roles in the tech industry with our nine possible job positions ranging from Software Engineer to Technical Sales. 
+      </p>
+      
+      <h5>How It Works</h5>
+      <p>
+        After registering on InterviewMe, you can play both the role of the interviewer or interviewee. 
+      </p> 
+      <h6>As an Interviewer</h6>
+      <p>
+        Be sure to update what positions you are qualified to give an interview for in your profile. This will be critical in matching you up with corresponding interviewees.
```
Comment (jalexanderqed) [inline src/main/webapp/about.html]: Optional suggestion: "to update the positions for which you are qualified to give an interview"

Comment (jalexanderqed) [inline src/main/webapp/about.html]: Break line

File: src/main/webapp/about.html [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+        <br>
+        We will use your availabilities specified on the <a href="availability.html">Availability Page</a> to schedule your interviews.
+        <br>
+        Interviewees will see your company and job when searching, so please update those accordingly. Once an interviewee is matched with you, you will get more details in an email. The interviews will be one hour long and hosted on Google Meet.
```
Comment (jalexanderqed) [inline src/main/webapp/about.html]: Break line

File: src/main/webapp/about.html [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+      </p> 
+      <h6>As an Interviewee</h6>
+      <p>
+        To search for an interview, navigate to the <a href="search-interview.html">Scheduling Page</a>. You can choose what position you're interested in and see what times and interviewers are available. You will be able to see their company and job, but no more personal information.
```
Comment (jalexanderqed) [inline src/main/webapp/about.html]: Break line

Comment (jalexanderqed) [inline src/main/webapp/about.html]: "you're" on this line is the only contraction on the page, which is a little jarring.

**Review by jalexanderqed — CHANGES_REQUESTED** [review]

File: src/main/webapp/about.html (hunk 4 above)
Comment (apluscs) [reply src/main/webapp/about.html]: l'm gonna break lines after the content gets approved, or else l'll have to redo a bunch of work (also in PR description)

Comment (jalexanderqed) [reply src/main/webapp/about.html]: Sorry for missing that

**Review by jalexanderqed — APPROVED** [review]

**Pull request merged**